src/test/java/com/coreos/jetcd/internal/impl/TestConstants.java

```diff
@@ -5,11 +5,11 @@
  */
 public class TestConstants {
 
-  public static final String[] endpoints = new String[]{
-      "http://localhost:2379", "http://localhost:22379", "http://localhost:32379"
-  };
+  public static final String[] endpoints = System.getProperty(
+      "etcd.endpoints",
```

Comment:
It can be used to provides different endpoints for testing so if ``etcd.endpoints`` is set as java system property, the test suite will be executed against such endpoints (i.e. I have my own etcd cluster and I want to use it for testing)

Same for ``etcd.peerurls``.